src/event_pusher/mod_event_pusher_http.erl

```diff
@@ -59,10 +59,10 @@ make_req(Host, Sender, Receiver, Message) ->
     Path = fix_path(list_to_binary(gen_mod:get_module_opt(Host, ?MODULE, path, ?DEFAULT_PATH))),
     PoolName = gen_mod:get_module_opt(Host, ?MODULE, pool_name, ?DEFAULT_POOL_NAME),
     Pool = mongoose_http_client:get_pool(PoolName),
-    EncodedHost = cow_qs:urlencode(Host),
-    EncodedSender = cow_qs:urlencode(Sender),
-    EncodedReceiver = cow_qs:urlencode(Receiver),
-    EncodedMessage = cow_qs:urlencode(Message),
+    EncodedHost = cow_qs:qs(Host),
```

Comment:
This is not a valid usage of `cow_qs:qs`. You may take a look at the unit test for inspiration. ;) https://github.com/ninenines/cowlib/blob/master/src/cow_qs.erl#L269